vsintegration/src/unittests/Tests.LanguageService.QuickInfo.fs

```diff
@@ -919,7 +919,7 @@ type QuickInfoTests() =
                 type A() =
                     static member Foo([<System.ParamArrayAttribute>] a : int[]) = ()
                 let r = A.Foo(42)""" ,
-            "type A","params a:"    )
+            "type A","[<ParamArray>] a:"    )
```

Comment:
Interesting.  In member intellisense, the module or namespace qualification is intelligently modified based on the current environment.

![image](https://cloud.githubusercontent.com/assets/5943573/6053485/d1cfb888-ac94-11e4-8ad6-3eefc2e2b8cf.png)

But yes, for hover tip on types, it seems that existing established behavior is to naively shorten, no matter what.

![image](https://cloud.githubusercontent.com/assets/5943573/6053498/f0e02aaa-ac94-11e4-97d3-18cce626f718.png)

So I guess the inconsistency is not worth worrying about.